cmd/zetae2e/README.md

```diff
@@ -57,14 +57,10 @@ contracts:
 ### Bitcoin setup
 Interaction with the Bitcoin node will require setting up a specific node tracking the address. It can be set locally following the guide [Using Bitcoin Docker Image for Local Development](https://www.notion.so/Using-Bitcoin-Docker-Image-for-Local-Development-bf7e84c58f22431fb52f17a471997e1f?pvs=21) 
 
-If an error occurs mention that wallets are not loaded. The following commands might need to be run in the Docker container:
+If an error occurs mention that wallets are not loaded, this command can be run to set it up:
```

Comment:
We should precise this is for local testing.
The command would not work on live networks since we use `GenerateToAddress`.
For live networks we can precise that we're expecting a node with a enable wallet to be already setup.